decoders/netflow/packet.go

```diff
@@ -114,6 +114,22 @@ type Field struct {
 
 	// The length (in bytes) of the field.
 	Length uint16
+
+	// IANA private enterprise number (PEN) of the authority defining the
+	// Information Element identifier in this Template Record.
```

Comment:
I'd add that it's only valid for IPFIX. Well, it's up to you.

Reply:
It's better to write clearly, thanks, added documention.